code/game/objects/items/weapons/pneumaticCannon.dm

```diff
@@ -88,9 +88,6 @@
 	if(!istype(user) && !target)
 		return
 	var/discharge = 0
-	if(is_in_gang(user, "Sleeping Carp"))
```

Comment:
There's more than this, I'm pretty sure; there's also some stuff under the projectiles module.